App/Containers/ThreadsList.js

```diff
@@ -216,9 +194,9 @@ const mapStateToProps = (state) => {
   return {
     profile,
     items,
-    showNotificationsPrompt: state.preferences.tourScreens.notifications && threads,
+    showNotificationsPrompt: state.preferences.tourScreens.notifications && items.length,
```

Comment:
Just curious about JS truthy stuff like this... so `&& items.length` is `true` if `length` > 0? Or what?

Reply:
But noticed below you actually compare with `=== 0`, so just want to be sure this was intentional.

Reply:
yeah, items.length > 0 => true.

i'm not certain this is going to behave perfectly, there are a few cases of this to ensure tour screens aren't shown just in the time before redux has rehydrated... would be nice to know the universal approach to doing that.